.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+
+Thank you for your Pull Request!  We have developed this task checklist from the [Development Process Guide](https://roche-gsk.github.io/admiral/articles/development_process.html) to help with the final steps of the process.  Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   
+
+Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request.  This checklist is part of the Github Action workflows and the Pull Request will not be merged into the devel branch until you have checked off each task.
```

Comment:
Please use a **single** space after the end of a sentence to be consistent with existing text in the package.

Reply:
Updated.  This is a a byproduct of the US educational system.